microhapulator/data/template.html

```diff
@@ -136,9 +162,45 @@ <h2>Read Mapping</h2>
                     <td class="alnrt">{{ "{:.2f}".format(row.MappingRate * 100) }}%</td>
                     <td class="alnrt">{{ "{:,}".format(row.MappedFullRefr) }}</td>
                     <td class="alnrt">{{ "{:.2f}".format(row.MappingRateFullRefr * 100) }}%</td>
+                    <td class="alnrt">{{ "{:.2f}".format(row.InterlocChiSq)}}</td>
                 </tr>
                 {% endfor %}
             </table>
+                <table id="mapping_rates_table" class="pagination_table">
```

Comment:
There is very little space between the table of per-sample mapping statistics and this new table of per-marker mapping statistics. At a minimum, we should add some vertical space for a clear separation between the two tables. But I think this new table would benefit from a bit more explanatory text of its own. Do you want to take a stab at that?

Reply:
Added some text and headers to make the separation more clear.

Reply:
I made some revisions. Let me know what you think.